Analytical_Engine/Query/Graph/Adjacency.cs

```diff
@@ -44,6 +44,12 @@ public static partial class Query
         [Output("adjacency", "The Dictionary where the keys are entities and the values are the collection of adjacent entities.")]
         public static Dictionary<Guid, List<Guid>> Adjacency(this Graph graph, RelationDirection relationDirection = RelationDirection.Forwards)
         {
+            if(graph == null)
+            {
+                BH.Engine.Reflection.Compute.RecordError("Cannot query the adjacency of a null graph.");
+                return null;
```

Comment:
Why not return an empty collection to follow the pattern?
```suggestion
                return new Dictionary<Guid, List<Guid>>();
```